### PR TITLE
Hotfix to avoid a crash downstream when optimal extraction fails

### PR DESCRIPTION
**Pull request by profxj**
As titled.

### CODE REVIEW

**Comment by profxj** [comment]
Note, the Travis failures are dealt with in `develop` 
and are not related to this PR.

**Review by rcooke-ast — APPROVED** [review]

File: pypeit/core/extract.py [hunk 1]
```diff
@@ -1917,8 +1917,6 @@ def ech_objfind(image, ivar, slitmask, slit_left, slit_righ, order_vec, maskslit
         except TypeError:
             std_in = None
```
Comment (jhennawi) [inline pypeit/core/extract.py:1919]: Please replace this ToDO. The problem still exists, and it is still thus a TODO.

**Review by jhennawi — APPROVED** [review]
TODO was reinserted since that problem in the code still exists.

**Comment by profxj** [comment]
merging

**Pull request merged**